app/views/admin/users/edit.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Admin Dashboard</h1>
```

Comment:
good to see we're in the 20's - this view has room to grow - simple but effective ^^